lib/cc/engine/analyzers/violation.rb

```diff
@@ -91,19 +112,15 @@ def fingerprint
           digest << "-"
           digest << current_sexp.mass.to_s
           digest << "-"
-          digest << occurrences.to_s
+          digest << occurrences.length.to_s
```

Comment:
With the change to reporting separate issues for each occurrence, this leads to some interesting behavior. Let's say I have identical code in 3 files & I copy-paste it to a fourth. The result will be that the previous 3 issues will be "fixed" and I'll have 4 "new" issues. If code in file A hasn't changed then it feels odd that the file A specific issue for the duplication got marked fixed and then new.

I'm not suggesting this should change, just thinking aloud abut whether this is the optimal choice on our part. Since all of the other locations are attached to each instance of the issue, it might be ok to not use occurrences as part of the fingerprint.